pilot/pkg/bootstrap/server.go

```diff
@@ -284,6 +299,19 @@ func (s *Server) initMixerSan(args *PilotArgs) error {
 	return nil
 }
 
+func (s *Server) getKubeCfgFile(args *PilotArgs) (kubeCfgFile string) {
```

Comment:
Maybe a small comment - I assume this return the primary kubecfg ?

Reply:
Yes, you are right.

Reply:
BTW - when running in k8s, I would expect the in-cluster config to be read as well, can you check this is the case ?

Reply:
Yes. line 317 makes sure that it'd work as before